test/integration/test/list.test.js

```diff
@@ -0,0 +1,18 @@
+const clientHelper = require('../src/client-helper');
+const configHelper = require('../src/config-helper');
+const RevAiApiJob = require('../../../dist/src/models/RevAiApiJob')
+
+test('Can get list of jobs', async () => {
```

Comment:
I would cluster the `list.test.js` and `job.test.js` in the same file `job.test.js` since the url is similar `/jobs`. 

Some interactions use POST, others use GET or DELETE.

Some require url params